unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PassportNavigation/PassportNavigationComponentView.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PassportNavigationComponentView : MonoBehaviour, IPassportNavigationComponentView
```

Comment:
In order to follow our UI Components architecture, all our views should implement the `BaseComponentView` class.

Reply:
Additionally, as we talked earlier in the meeting, maybe it would be useful to make this view to be part of our common UI Components library.